src/workspaces/list/WorkspaceFilters.ts

```diff
@@ -32,14 +32,14 @@ export const WorkspaceFilters = (props: WorkspaceFiltersProps): ReactNode => {
   const { query } = Nav.useRoute();
   const filters = getWorkspaceFiltersFromQuery(query);
 
-  let keywordLastEvented = useInstance(() => filters.nameFilter);
+  let keywordLastEvented = useInstance(() => filters.keywordFilter);
   const [lastKeywordSearched, setLastKeywordSearched] = useState(keywordLastEvented);
 
   return div({ style: { display: 'flex', margin: '1rem 0' } }, [
     div({ style: { ...styles.filter, flexGrow: 1.5 } }, [
       h(DelayedSearchInput, {
-        placeholder: 'Search by keyword',
-        'aria-label': 'Search workspaces by keyword',
+        placeholder: 'Search by name, project, or state',
```

Comment:
Making "project" purposefully vague because both billing project (namespace, existing behavior) and googleProject (new) will be searched.